src/foam/dao/EasyDAO.js

```diff
@@ -162,6 +188,13 @@ if ( getContextualize() ) {
     build();
 }
 
+if ( getOrder() != null &&
+     getOrder().length > 0 ) {
```

Comment:
The 'order' property isn't an array, so .length doesn't make sense.

Reply:
@kgrgreer 'order' is an FObjectArray of foam.mlang.order.Comparator

Reply:
'order' is an array, the issue is the dao.orderBy call which needs to compose them in some way.